<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutorizationForm</class>
 <widget class="QDialog" name="AutorizationForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QComboBox" name="m_usersList">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>21</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>21</y>
     <width>112</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Пользователь:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>54</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_password">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>54</y>
     <width>142</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>452</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>240</number>
    </property>
    <item>
     <widget class="QPushButton" name="m_OKButton">
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_cancelButton">
      <property name="text">
       <string>Отмена</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>90</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Роль:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_roleList">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxCount">
    <number>2147483646</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
